### PR TITLE
Never clone forks

### LINKED ISSUE #59: Fork issue
For some reason we ended up with a fork https://github.com/ropensci/climatrends that turned up in the registry, but this package has not been reviewed AFAICT.

Should our script simply remove all forks / not clone them?

### PR DESCRIPTION
**Pull request by maelle**
Fix #59

### CODE REVIEW

File: inst/automation/git_urls.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@ def gh_default_branch(owner_repo)
     "package" => repo["name"],
     "url" => repo["html_url"],
     "branch" => repo["default_branch"]
-  } unless repo["archived"] || ex.include?(repo["name"])
+  } unless repo["archived"] || repo["fork"] || ex.include?(repo["name"])
```
Comment (maelle) [inline inst/automation/git_urls.rb:65]: seems harmless? :sweat_smile:

**Comment by maelle** [comment]
I've opened #61

**Comment by jeroen** [comment]
I don't speak any ruby either, so I guess we'll just test it and see if it works.

**Comment by jeroen** [comment]
Almost worked: https://github.com/ropensci/roregistry/commit/614332c689430d59e906067c18da08820a04b76c

**Comment by maelle** [comment]
It didn't work for the registry though so I'll have to dig a bit more.

**Pull request merged**